docs/sources/next/extensions/explore.md

```diff
@@ -35,10 +35,6 @@ Use the table to explore the many extensions. Questions? Feel free to join the d
         <h4>xk6-ansible-vault</h4>
         <p>Encrypt and decrypt Ansible Vault</p>
     </a>
-    <a href="https://github.com/joanlopez/xk6-aws" target="_blank" class="nav-cards__item nav-cards__item--guide">
```

Comment:
The extension does not meet the registry requirements.
https://github.com/joanlopez/xk6-aws/issues/2
https://github.com/joanlopez/xk6-aws/issues/1